Src/Fido2.BlazorWebAssembly/Fido2.BlazorWebAssembly.csproj

```diff
@@ -39,4 +39,8 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="Microsoft.SourceLink.GitHub" Version="8.0.0" />
```

Comment:
Do we need this package reference? Starting with .NET 8, Source Link is included in the .NET SDK and enabled by default: https://github.com/dotnet/sourcelink/?tab=readme-ov-file#using-source-link-in-net-projects

Reply:
Probably not. Let me see if I can remove it.